### DISCUSSION TITLE
Stored Procedure 의 장단점은 무엇일까?

### DISCUSSION THREAD

**Discussion by leehosung** [8percent/rdb-study#41] [Ideas]
고민 해 보자.

**Comment by kyungjunleeme** [comment]
제갈윤님이 언급해주셨던 배달의 민족에서 과거 Stored Procedure를 사용했던 사례에 관한 영상입니다.

https://www.youtube.com/watch?v=BnS6343GTkY

관심있으신분들 참고하시면 될 것 같습니다.
